core/electron-manager/src/ElectronManager.ts

```diff
@@ -165,7 +180,9 @@ export class WebpackDevServerElectronManager extends ElectronManager {
 // this initialization should always happen before the app starts.
 function staticElectronInitialize() {
   app.allowRendererProcessReuse = true; // see https://www.electronjs.org/docs/api/app#appallowrendererprocessreuse
-  protocol.registerSchemesAsPrivileged([{ scheme: "electron", privileges: { standard: true, secure: true } }]);
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  if (!app.isReady)

```

Comment:
@wgoehrig This is necessary because certa loads electron before we have a chance to do anything, so "isReady" is true before any of our code gets loaded. Due to their crazy api that requires this step be completed before the app is ready, we either have to do this in certa or stop relying on it (at least in tests). So far it doesn't seem to be used in our tests.

Reply:
Yeah, if tests aren't using it, I think it's probably fine to stop relying on it in tests...

I'll look into whether there's a way for certa to load this before the app "isReady" though...